packages/@aws-cdk/aws-rds/test/integ.instance.lit.ts

```diff
@@ -26,9 +26,7 @@ class DatabaseInstanceStack extends cdk.Stack {
 
     /// Add XMLDB and OEM with option group
```

Comment:
curious - do we still follow this `.lit.*` naming convention? I haven't been doing it but would love to know if it's something I *should* be doing

Reply:
It's just for files we reference from the ReadMe files - they are included verbatim into the rendered ReadMe.

Reply:
aside (not for this PR): didn't realize we did that.

I think we should stop referencing the `.lit` files as it doesn't render in `npm` and presumably other package managers

 - i.e. [`Certificate Manager`](https://www.npmjs.com/package/@aws-cdk/aws-certificatemanager) leads off with an example - but clicking on that link hits me with a `404` - https://github.com/aws/aws-cdk/blob/HEAD/test/example.simple-request.lit.ts

there's a good 20+ modules where we are doing this, but we should reconcile as a chore or something. happy to do it if you agree!